File: classics/fizzbuzz/fizzbuzz.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-

for n in range(1, 101):
    if n % 15 == 0:
        print("fizzbuzz")
    elif n % 5 == 0:
        print("buzz")
    elif n % 3 == 0:
        print("fizz")
    else:
        print(n)

# EOF

File: classics/fizzbuzz/fizzbuzz.go
//usr/bin/env go run "$0" "$@"; exit "$?"

package main

import "fmt"

func main() {
    for n := 1; n < 101; n++ {
        if n % 15 == 0 {
            fmt.Println("fizzbuzz");
        } else if n % 5 == 0 {
            fmt.Println("buzz");
        } else if n % 3 == 0 {
            fmt.Println("fizz");
        } else {
            fmt.Println(n);
        }
    }
}

// EOF
